protocol/x/ratelimit/types/expected_keepers.go

```diff
@@ -15,3 +18,23 @@ type BankKeeper interface {
 type BlockTimeKeeper interface {
 	GetTimeSinceLastBlock(ctx sdk.Context) time.Duration
 }
+
+// ICS4Wrapper defines the expected ICS4Wrapper for middleware
+type ICS4Wrapper interface {
```

Comment:
noob question: does this interface exist somewhere already like ibc-go? if so, why do we need to re-define it here?

Reply:
The interface does, but here we only define the subset of the interface used by the `ratelimitKeeper`. It's the same pattern as all other keeper interfaces defined within this `expected_keepers.go` file